working/v3.0.0-draft2/ciba-flow/CIBA-login-hint-token-schema.json

```diff
@@ -0,0 +1,64 @@
+{
+  "definitions": {},
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "http://apicentre.paymentsnz.co.nz/schemas/CIBA-login-hint-schema.json",
+  "type": "object",
+  "title": "The schema for the CIBA request object login_hint_token",
+  "additionalProperties": false,
+  "required": ["subject"],
+  "properties": {
+    "subject": {
+      "$id": "/properties/subject",
+      "type": "object",
+      "title": "Subject of the token",
+      "required": ["subject_type"],
+      "maxProperties": 2,
+      "minProperties": 2,
+      "properties": {
+        "subject_type": {
+          "$id": "/properties/subject/properties/subject_type",
+          "type": "string",
+          "title": "The type of subject hint",
+          "enum": [
+            "phone",
+            "email",
+            "username",
+            "api_provider_token",
+            "third_party_token"
+          ]
+        },
+        "phone": {
+          "$id": "/properties/subject/properties/phone",
+          "type": "string",
+          "title": "The phone of the subject identity to authorise",
+          "examples": ["+64-21-101-1221"]
+        },
+        "email": {
+          "$id": "/properties/subject/properties/email",
+          "type": "string",
+          "title": "The email of the subject identity to authorise",
+          "format": "email",
+          "examples": ["user@example.com"]
+        },
+        "username": {
+          "$id": "/properties/subject/properties/username",
+          "type": "string",
+          "title": "The username of the subject identity to authorise",
+          "examples": ["user.name", "@handle"]
+        },
+        "api_provider_token": {
+          "$id": "/properties/subject/properties/api_provider_token",
+          "type": "string",
+          "title": "A token generated by the customer authentication device",
+          "description": "Uniquely identifies the Customer authentication device and Customer with the API Provider"
+        },
+        "third_party_token": {
+          "$id": "/properties/subject/properties/third_party_token",
+          "type": "string",
+          "title": "A token generated by the third party and registered with the customer authentication device",
+          "pattern": "^[A-Za-z0-9-_]+\\.[A-Za-z0-9-_]+\\.[A-Za-z0-9-_.+/=]*$"
```

Comment:
I couldn't find any reference where this is required to be JWT-like separated by periods. Is this the case?

Reply:
In this case, the Security Profile isn't terribly clear.  However, in general - all 'tokens' (used as hints in CIBA) are expected to be signed JWT tokens, so the pattern is correct.